db/migrate/20210108094047_rename_brexit_checker_app_name.rb

```diff
@@ -0,0 +1,8 @@
+class RenameBrexitCheckerAppName < ActiveRecord::Migration[6.0]
+  def up
+    Doorkeeper::Application.find_by(uid: "r0JFn7O_750_W4i-IjQpMA8j970dNchgKbJ7ZGz0ozI")&.update!(name: "Brexit checker (Demonstration App)")
```

Comment:
Which environment is this in?

Reply:
Yeah odd one right? I was scratching my head about it too.
It's on staging right now. Is it a remenant of an old integration env?

Prod only has the last one.